backend/server/routers/user.py

```diff
@@ -1,19 +1,62 @@
+from os import getenv
+from typing import cast
 from fastapi import APIRouter, HTTPException
-from server.database import usersDB
+from server.database import create_dynamic_db, usersDB
 from bson.objectid import ObjectId
-import pydantic 
+from server.routers.model import LocalStorage, Storage
+import pydantic
+from server.config import DUMMY_TOKEN
 pydantic.json.ENCODERS_BY_TYPE[ObjectId]=str
 
 router = APIRouter(
     prefix="/user",
     tags=["user"],
 )
 
+# DANGEROUS!!!!!!!
+# NEVER ALLOW TO RUN IN PROD
+# PLEASE DONT BE STUPID
+if getenv("DANGEROUS_ALLOW_DELETE_DB_REQUEST") == "true":
+    @router.post("/drop")
```

Comment:
Let's maybe not have something like this possibly run in production; it would be a better idea to have your tests directly execute MongoDB queries rather than relying on an endpoint like this 😄 

I will re-review once changes that were discussed on Discord are made (env vars for testing)